app/mailers/approval_notifier_mailer.rb

```diff
@@ -23,18 +23,37 @@ def reset_password_approval_email(user_id:)
 
   private
 
+  # permissions_users are admins as per the old permissions system
+  # accesses_users are admins as per the new permissions system
+  # we need to send emails to the superset of the two till we flip everyone over to new permissions
+
   def supervisor_emails
-    users = UserPermission.where(permission_slug: :approve_health_workers, resource: user.facility_group).map(&:user)
+    permissions_users = UserPermission.where(permission_slug: :approve_health_workers, resource: user.facility_group).map(&:user)
+    accesses_users = User.admins.manager_access
+      .select { |admin| admin.accessible_facilities(:manage).include?(user.facility) }
+      .reject { |admin| admin.accesses.map(&:resource).include?(user.organization) }
```

Comment:
@vkrmis Is this different from,

```ruby
admin.accessible_organizations(:manage).where(id: user.organization).any?
```

...this?

Reply:
Yes, `supervisor_emails` are `facility` and `facility_group` managers, so `accessible_organizations(:manage)` doesn't work. It would work for `organization_owner_emails` though, I'll change it there.